### PR TITLE
Feature: banner hotfix

### LINKED ISSUE #5110: Banner button link regression
# Problem

The recent banner link title update has caused a regression that is outputting an empty button.  

Screen Shot 2022-04-07 at 5 00 10 PM

# Steps to reproduce

https://translation.uiowa.edu

# Proposed solution

Revisit template logic so that link button does not output if there is a title, and link without link text.

### PR DESCRIPTION
**Pull request by bspeare**
Resolves https://github.com/uiowa/uiowa/issues/5110

# How to test

## Banner link

- Create a banner with a title and no link
  -  No buttons should appear
- Create a banner a title and just a link url
  - No button should appear
- Create a banner without a title and a link url
   - You should see a button render 
- Create multiple links
  - You should see multiple buttons

## Pre-title
- Add a pre-title with an empty title
  - The pre-title should not render
- Add a pre-title with a banner title
  - The title and pre-title should render

### CODE REVIEW

**Comment by joewhitsitt** [comment]
pulling to test

**Comment by bspeare** [comment]
@joewhitsitt adding one more test description in a minute.

**Comment by joewhitsitt** [comment]
Sorry, I don't think I was quite awake and didn't notice you hadn't requested a review yet. I saw hotfix and jumped in my seat. If this PR is ready, I think it checks out. 👍

**Comment by pyrello** [comment]
I believe that the change here isn't quite right. If the title exists and there is a link with text, then the title should be the linked element and the link should be rendered as a pseudo-button.

File: docroot/themes/custom/uids_base/templates/uids/banner.html.twig [hunk 1]
```diff
@@ -90,7 +89,7 @@
                   {{ link.link_text }}
                   <i role="presentation" class="fas fa-arrow-right"></i>
                 </a>
-              {% else %}
+              {% elseif link_element == 'title' %}
                 {# Display as a pseudo-link. #}
                 <div aria-hidden="true" {{ button_attributes.addclass(link.link_classes) }} {% if banner_aria_describedby %} id="{{ banner_aria_describedby }}" {% endif %}>
                   {{ link.link_text }}
```
Comment (joewhitsitt) [inline docroot/themes/custom/uids_base/templates/uids/banner.html.twig:100]: does this just need to be set to banner_title?

Comment (joewhitsitt) [reply docroot/themes/custom/uids_base/templates/uids/banner.html.twig:100]: wait, that just reprints the banner title as the button text. i don't think that is the desired result

**Review by richardbporter — APPROVED** [review]
Test criteria checked out.
Screen Shot 2022-04-08 at 1 30 59 PM

Also synced translation.uiowa.edu and the button is not rendering there.
Screen Shot 2022-04-08 at 1 32 33 PM

**Comment by pyrello** [comment]
Thanks for testing!

**Pull request merged**